vars/s3PublishArtifacts.groovy

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/groovy
+
+def call(Map params) {
+
+    final String bucket = params.bucket
+    final String filePattern = params.filePattern ?: '*.*'
+    final String directory = params.directory ?: '.'
+
+    dir(directory) {
+        def files = findFiles(glob: filePattern)
```

Comment:
Is this a `List<String>` or `List<File>` or something else, or does it matter?

Reply:
I'm not sure exactly, it was copied directly from our release pipeline, docs for this step are here https://jenkins.io/doc/pipeline/steps/pipeline-utility-steps/#code-findfiles-code-find-files-in-the-workspace